source/components/atoms/RadioButton/RadioButton.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { TouchableHighlight, LayoutAnimation } from 'react-native';
+import styled, { ThemeContext } from 'styled-components';
+import SHADOW from '../../../styles/shadow';
+
+const TouchableSizes = {
+  small: {
+    height: 22,
+    width: 22,
+    borderRadius: 11,
+  },
+  medium: {
+    height: 36,
+    width: 36,
+    borderRadius: 18,
+  },
+  large: {
+    height: 48,
+    width: 48,
+    borderRadius: 24,
+  },
+};
+
+const BorderSizes = {
+  small: {
+    height: 24,
+    width: 24,
+    borderRadius: 12,
+    borderWidth: 2,
+    padding: 0,
+    margin: 0,
+  },
+  medium: {
+    height: 36,
+    width: 36,
+    borderRadius: 18,
+    borderWidth: 3,
+    padding: 0,
+    margin: 0,
+  },
+  large: {
+    height: 48,
+    width: 48,
+    borderRadius: 24,
+    borderWidth: 4,
+    padding: 0,
+    margin: 0,
+  },
+};
+
+const FillSizes = {
+  small: {
+    height: 16,
+    width: 16,
+    borderRadius: 10,
+    margin: 2,
+  },
+  medium: {
+    height: 25,
+    width: 25,
+    borderRadius: 12.5,
+    margin: 4,
+  },
+  large: {
+    height: 35,
+    width: 35,
+    borderRadius: 17.5,
+    margin: 6,
+  },
+};
```

Comment:
Would be good to move these into the theme.js, also check out if you could utalize the sizes in the setLayout function in theme.js (y)

Reply:
Moved into theme.js